config.js

```diff
@@ -1,28 +1,30 @@
 function load(envName) {
-    const envOverrides = {
-        'production': () => require('./config/production.json'),
-        'qa': () => require('./config/qa.json'),
-        'development': () => require('./config/development.json'),
-    };
-
-    const baseConfig = require('./config/base.json');
-    const overrideConfig = envOverrides.hasOwnProperty(envName) ? envOverrides[envName]() : {};
-
     return {
-        ...baseConfig,
-        ...overrideConfig
+        ...require('./config/base.json'),
+        ...loadConfig(envName),
+        ...loadConfig('local'),
```

Comment:
```suggestion
        ...require('./config/base.json'),
        ...loadConfig(envName),
```

Thoughts on treating `local.json` like any other `envName`? I can see the possibility of someone having a `local.json` file overwriting (I believe order of object expansion has overwrite rules) or adding undesired config values. The process can be outlined/updated in the README.

Reply:
It think local should be special in that you can run any config you want and override some values without checking them in to version control.

I don't foresee it being very useful, but sometimes people have wonky setups and it should help with community setup of dev environments

Reply:
Ah, so local isn't a full environment? Just an override file?

Reply:
that's the intent, yeah